components/TransactionForm.js

```diff
@@ -0,0 +1,106 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+export default function TransactionForm({ onAddEntry, title }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    onAddEntry(data);
+  }
+
+  const router = useRouter();
```

Comment:
Please see my comments in the suggestion below. 🙃

```suggestion
import { useRouter } from "next/router";
// import the next/link component - and change the `<StyledCancelButton> - button` to a Link 
import Link from 'next/link';
import styled from "styled-components";

export default function TransactionForm({ onAddEntry, title }) {
  function handleSubmit(event) {
    event.preventDefault();

    const formData = new FormData(event.target);
    const data = Object.fromEntries(formData);

    onAddEntry(data);
    // as mentioned in the comment below, call `router.push()` after ` onAddEntry(data)`
    router.push('./overview');
  }


```